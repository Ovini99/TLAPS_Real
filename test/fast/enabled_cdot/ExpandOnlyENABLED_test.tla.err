@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:10:1:10:3
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:10:4:10:17
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:2
@!!END

File "./ExpandOnlyENABLED_test.tla", line 9, characters 18-32:
Error: Nested occurrence of the operator `\cdot` found in scope of `ENABLED` when expanding `ENABLED`. Using the proof directive `ExpandCdot` from the module `TLAPS` is expected to avoid this error.For example, writing `BY ExpandCdot`.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:10:1:10:3
@!!status:failed
@!!prover:expandenabled
@!!meth:time-limit: inf
@!!reason:Expr.Action
@!!already:false
@!!obl:Expr.Action
Obligation:

ASSUME TRUE 
PROVE  ENABLED (TRUE \cdot TRUE)

@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:10:4:10:17
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

File "./ExpandOnlyENABLED_test.tla", line 10, characters 1-2:
[ERROR]: Could not prove or check:
           ASSUME ExpandENABLED 
           PROVE  ENABLED (TRUE \cdot TRUE)
File "./ExpandOnlyENABLED_test.tla", line 1, character 1 to line 11, character 80:
[ERROR]: 1/2 obligations failed.
There were backend errors processing module `"ExpandOnlyENABLED_test"`.
