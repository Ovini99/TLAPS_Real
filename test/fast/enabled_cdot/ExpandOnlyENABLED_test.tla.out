
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:14:22 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 --nofp ExpandOnlyENABLED_test.tla --stretch 5

(* created new ".tlacache/TLAPS.tlaps/TLAPS.thy" *)
(* fingerprints written in ".tlacache/TLAPS.tlaps/fingerprints" *)
(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from the file `.tlacache/ExpandOnlyENABLED_test.tlaps/fingerprints`, in order to overwrite with the new fingerprints, and then save the results at the end. *)
(* created new ".tlacache/ExpandOnlyENABLED_test.tlaps/ExpandOnlyENABLED_test.thy" *)
(* fingerprints written in ".tlacache/ExpandOnlyENABLED_test.tlaps/fingerprints" *)
